config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits-release-0-10.yaml

```
@@ -0,0 +1,44 @@
+presubmits:
+  kubernetes-sigs/node-feature-discovery:
+  - name: pull-node-feature-discovery-verify-release-0-10
+    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$"
+    branches:
+    - ^release-0.10
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-discovery
+      testgrid-tab-name: verify-release-0-10
```

nit/question: could we use dots at least in the tab names(?) Would make the testgrid slightly easier to read. WDYT?

```suggestion
      testgrid-tab-name: verify-release-0.10
```